bootstrap/src/main.ts

```diff
@@ -44,6 +26,38 @@ try {
   console.error(error);
 }
 
+/* Minor code duplication of StartURLManager, but Typescript gives compile problems we can't easily ignore when we import it */
+function processApp2AppArgs(url?: string): void {
+  const queryString = url || location.search.substr(1);
+  let pluginId, windowManager;
+
+  queryString.split('&').forEach(part => {
+    const [key, value] = part.split('=').map(v => decodeURIComponent(v));
+    switch (key) {
+      case "pluginId":
+        (window as any)['GIZA_PLUGIN_TO_BE_LOADED'] = value;
+        (window as any)['GIZA_SIMPLE_CONTAINER_REQUESTED'] = true;
+        (window as any)['GIZA_ENVIRONMENT'] = 'MVD';
+        pluginId = value;
+        break;
+      case "showLogin":
+        (window as any)['ZOWE_SWM_SHOW_LOGIN'] = value;
+        break;
+      case "windowManager":
+        windowManager = value;
+        break;
+    }
+  });
+
+  if ((window as any)['GIZA_SIMPLE_CONTAINER_REQUESTED']) {
+    if (windowManager == 'MVD' || windowManager == 'mvd') {
```

Comment:
You could do `windowsManager windowManager.toUpperCase() === 'MVD'`

Reply:
I didn't do it this way because TS gets weird

![image](https://user-images.githubusercontent.com/20528015/99999813-b62de880-2d8e-11eb-9441-00de7bc15da3.png)

but I can go around this by setting windowManager to be of type any